js/battle.js

```diff
@@ -1729,11 +1729,12 @@ var Side = (function () {
 			this.pokemon.push(poke);
 		}
 		if (replaceSlot === -2) this.battle.maxPokemon = this.pokemon.length;
-		if (this.pokemon.length > this.battle.maxPokemon || this.battle.speciesClause) {
```

Comment:
Why though?

Reply:
Keeping `this.battle.speciesClause` broke team preview. I'll see if I can solve that some other way...

Reply:
...why though?

Reply:
From what I could tell: because every team preview tier had it as false before, so when Pokemon got added with `|poke|` through team preview and `this.pokemon.length === this.battle.maxPokemon`, it would skip this check because it was false. When default is true (from now), and every searchid still is `''` it get loads of false positives.

Reply:
`if (!poke1.searchid) continue;` may or may not have already fixed that, though, but it might also have other side effects?

Reply:
It should skip searchids that are `''` though

Reply:
That's not necessarily the right solution...

Reply:
I thought it through and I don't think it'd have other side effects.